bin/plugin/restricted/groupCreate

```diff
@@ -83,6 +83,14 @@ if (!$group || !$owner) {
     osh_exit 'ERR_MISSING_PARAMETER', "Group name or owner is missing";
 }
 
+# first, check that the namme doesn't start with 'key' (see https://github.com/ovh/the-bastion/issues/178)
```

Comment:
```suggestion
# first, check that the name doesn't start with 'key' (see https://github.com/ovh/the-bastion/issues/178)
```

Reply:
Damn, I should really spend some time to dig why my keyboard is so prone to double-typing on some keys :(